.github/workflows/cloudbeat-ci.yml

```diff
@@ -114,104 +114,154 @@ jobs:
       matrix:
         include:
           - test-target: pre_merge
```

Comment:
If there is some easy workaround to avoid hardcoding kind-mono as the default on multiple lines, would be a nice plus.

Reply:
It can be done, as in [this](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations) example.
So we need to add more steps to create an environment variable if the `${{ matrix.kind }}` is not set. I like it more explicitly. Wdyt?